packages/ui/src/machines/authenticator/index.ts

```diff
@@ -170,7 +171,10 @@ export function createAuthenticatorMachine() {
             },
           },
           on: {
-            'done.invoke.signOutActor': 'signIn',
+            'done.invoke.signOutActor': [
+              { target: 'setup', cond: 'shouldSetup' },
+              { target: 'setup.goToInitialState' },
```

Comment:
If we have already setup, we go to `setup.waitConfig` to get `INIT` event from Authenticator. Otherwise, we skip setup and go directly to `initialState`.